app/lib/controlled_work_mappings/cw1_new.yml

```diff
@@ -0,0 +1,441 @@
+
+    # Main home / client's share of total net equity (non-SMOD)
+    - name: undefined_34
+      type: text
+      source: non_smod_main_home_net_equity
+
+    # Other property / client's share of total net equity (non-SMOD)
+    - name: undefined_35
+      type: text
+      source: non_smod_additional_properties_net_equity
+
+    # Main home / final assessed amount of client's equity (non-SMOD)
+    - name: undefined_36
+      type: text
+      source: non_smod_main_home_assessed_equity
+
+    # Other property / final assessed amount of client's equity (non-SMOD)
+    - name: undefined_37
+      type: text
+      source: non_smod_additional_properties_assessed_equity
+
+    # Savings (non-SMOD)
+    - name: undefined_38
+      type: text
+      source: non_smod_client_savings
+
+    # Investments (non-SMOD)
+    - name: undefined_40
+      type: text
+      source: non_smod_client_investments
+
+    # Valuables (non-SMOD)
+    - name: undefined_42
+      type: text
+      source: non_smod_client_valuables
+
+    # Total capital part A (non-SMOD)
+    - name: undefined_46
+      type: text
+      source: combined_non_disputed_capital
+
+    # Total of all capital assets (Part A and Part B)
+    - name: undefined_47
+      type: text
+      source: combined_assessed_capital
+
+- section: partner_capital
+  fields:
+    # Partner savings (non-SMOD)
+    - name: undefined_39
+      type: text
+      source: partner_savings
+
+    # Partner investments (non-SMOD)
+    - name: undefined_41
+      type: text
+      source: partner_investments
+
+    # Partner valuables (non-SMOD)
+    - name: undefined_43
+      type: text
+      source: partner_valuables
+
+- section: income_even_if_passported
+  fields:
+    # Part C
+    # The client is directly or indirectly in receipt of [passporting benefit]
+    - name: Passported
+      type: boolean_radio
+      source: passporting
+      section: income_even_if_passported
+      yes_value: "Yes"
+      no_value: "No"
+
+- section: income
+  fields:
+
+    # Gross monthly earnings / Client
+    - name: undefined_48
+      type: text
+      source: client_gross_income
+
+    # Maintenance from former partner / Client
+    - name: maintenance_client
+      type: text
+      source: client_maintenance_in
+
+    # Pensions / Client
+    - name: pensions_client
+      type: text
+      source: client_pensions
+
+    # Student Finance / Client
+    - name: studentfinance_client
+      type: text
+      source: client_student_finance
+
+    # Family or Friends / Client
+    - name: friendsandfamily_client
+      type: text
+      source: client_friends_and_family
+
+    # Property or Lodger / Client
+    - name: property_client
+      type: text
+      source: client_property_lodger_income
+
+    # Other Income / Client
+    - name: otherincome_client
+      type: text
+      source: client_other_income
+
+    # Benefits & Tax Credits / Client
+    - name: benefits_client
```

Comment:
Could we move `benefits_client` to be above `maintenance_client` so it follows the layout of the PDF - just makes it easier to read?